### PR TITLE
Update Rails and JS dependencies

### PR DESCRIPTION
**Pull request by ijdickinson**
- Fix sundry Rubocop warnings
- Update Ruby dependencies
- Update Javascript dependencies
- Update version and changelog

### CODE REVIEW

File: app/presenters/landing_state.rb [hunk 1]
```diff
@@ -75,7 +75,7 @@ def to_value(val)
   def format_percentage(change)
     if change == 'unknown'
       change
-    elsif change == 0.0
+    elsif change.abs <= 0.0001
```
Comment (joescottdave) [inline app/presenters/landing_state.rb:78]: What's this change for?

**Review by joescottdave — COMMENTED** [review]
Bit confused by the 0.0 => 0.0001 change
Looks good otherwise.

File: app/presenters/landing_state.rb (hunk 1 above)
Comment (ijdickinson) [reply app/presenters/landing_state.rb:78]: Rubocop was whining that exact comparisons (`==`) on floating point numbers are unreliable. While that is true for floating point in general, it shouldn't be true for `0.0`. So a better change might have been to `rubocop:disable` that check on that line

File: app/presenters/landing_state.rb [hunk 2]
```diff
@@ -75,7 +75,7 @@ def to_value(val)
   def format_percentage(change)
     if change == 'unknown'
       change
-    elsif change == 0.0
+    elsif change.zero?
```
Comment (joescottdave) [inline app/presenters/landing_state.rb:78]: 😄

**Review by joescottdave — APPROVED** [review]
Lgtm

**Pull request merged**